Vagrantfile

```diff
@@ -2,34 +2,45 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-	config.vm.box = "ubuntu/xenial64"
-    config.vm.network "private_network", type: "dhcp"
+    config.vm.box = "ubuntu/xenial64"
+
+    $script = <<-SCRIPT
+        touch /etc/vagrantvm
 
-    $suscript = <<-SCRIPT
         echo "deb http://apt.postgresql.org/pub/repos/apt/ xenial-pgdg main" > /etc/apt/sources.list.d/pgdg.list
         wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
-        sudo add-apt-repository -y ppa:deadsnakes/ppa
+        add-apt-repository -y ppa:deadsnakes/ppa
 
         apt-get update
         apt-get install -y build-essential libsnappy-dev
+
+        sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/g' /etc/ssh/sshd_config
+        systemctl reload ssh
+        useradd -m -U u12345678
+        echo "u12345678:password" | chpasswd
```

Comment:
I'm not sure how the networking is set up and can you actually make external connections but this would in any event look less suspicious if password was derived from urandom and stored in some file from which it can be read locally in tests.